CIAOS-Rust/client/ciaos/client.py

```diff
@@ -0,0 +1,183 @@
+# client.py
+
+import sys
+import os
+sys.path.append(os.path.join(os.path.dirname(__file__), 'util'))
+import requests
+from typing import List
+from util.flatbuffer.flatbuffer_handler import create_flatbuffer, parse_flatbuffer
+
+
+def put(user: str, api_url: str, key: str, data_list: List[bytes]):
```

Comment:
Where is the interface, similar to boto?

```
put(key,filepath,dataList)
```

If they give filepath, just upload the file else use datalist